doc/how_to/unsigned_to_signed.rst

```diff
@@ -0,0 +1,82 @@
+Unsigned to Signed Data types
+=============================
+
+As of version 0.103.0 SpikeInterface has changed one of its defaults for interacting with
+:code:`Recording` objects. We no longer autocast unsigned dtypes to signed implicitly. This
+means that some users of SpikeInterface will need to add one additional line of code to their scripts
+to explicitly handle this conversion.
+
+
+Why this matters?
+-----------------
+
+For those that want a deeper understanding of dtypes `NumPy provides a great explanation <https://numpy.org/doc/stable/reference/arrays.dtypes.html>`_.
+For our purposes it is important to know that many pieces of recording equipment opt to store their electrophysiological data as unsigned integers,
+which provides the benefit of reducing the necessary file size. In order to convert to real units these file formats only need to store a :code:`gain`
```

Comment:
I don't think this is true, since `unit16` and `int16` has the same size (16 bit). The difference is that unsigned values have a twice as large range ov values, because for signed ints the first bit is the sign.